assets/js/touchpoints_translations.js

```diff
@@ -1,13 +1,20 @@
-const yesButton = document.querySelector('input[type=submit][value=yes]');
-const noButton = document.querySelector('input[type=submit][value=no]');
+/**
+ * Attached to Touchpoints script tag as an onload callback
+ */
+function translateTouchpoints() {
+  const yesButton = document.querySelector('input[type=submit][value=yes]');
+  const noButton = document.querySelector('input[type=submit][value=no]');
```

Comment:
this PR is just a proof-of-concept, not necessarily ready to be merged, I'd prefer a more dynamic translation approach like is suggested here: https://github.com/18F/identity-site/pull/636#discussion_r638229566